arangod/Cluster/AgencyCache.cpp

```diff
@@ -470,3 +540,114 @@ void AgencyCache::invokeAllCallbacks() const {
   }
   invokeCallbacks(toCall);
 }
+
+
+//TODO Snapshot case to do
+AgencyCache::change_set_t const AgencyCache::changedSince(
```

Comment:
I think the const has no effect, unless you make it a const reference. But I guess this function should return by value and not by reference?
```suggestion
AgencyCache::change_set_t AgencyCache::changedSince(
```